tsconfig.json

```diff
@@ -2,9 +2,12 @@
 {
   "compilerOptions": {
     /* Basic Options */
-    "target": "es3",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
+    "target": "es5",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
```

Comment:
Switching to ES5 might 'screw' some older browsers. Is there a clear benefit we should move to ES5?

SIde note. We do not actively support IE < 10 browsers, and for other modern browsers: FF, Chrome, Safari we support the current and previous version. However. If without too much effort we can support other/older browsers. We should never refrain from doing so. As some institutions still use ancient browsers and supporting them with _minimal_ effort should always be attempted. And, we should never put our-self at risk by supporting older browsers.

Reply:
> Switching to ES5 might 'screw' some older browsers. Is there a clear benefit we should move to ES5?
Projects like Webauthn also use es5. ES5 is already more than 10 years old. Older browser are not safe anyway so i think we should not support that. Besides that, Engineblock has support for IE10+ which is similar as for ES5.

Reply:
Thanks for verifying this!